src/components/footer/_macro.njk

```diff
@@ -157,11 +173,12 @@
                             </div>
                         {% endif %}
                         {% if not params.poweredBy %}
-                            <a class="ons-footer__poweredBy-link" {% if lang == "cy" %}href="https://cy.ons.gov.uk/"{% else %}href="https://www.ons.gov.uk/"{% endif %}>
-                                <div class="ons-footer__poweredby-logo ons-u-mb-m">
-                                    {{ onsLogo | safe }}
+                                <div class="ons-footer__poweredby-logo ons-u-mb-m{% if additionalLogo %} ons-grid--flex{% endif %}">
```

Comment:
```suggestion
                                <div class="ons-footer__poweredby ons-u-mb-m{% if additionalLogo %} ons-grid--flex{% endif %}">
```

We can now rename this class as its no longer always just one logo. Other than this looks good